protoc-artifacts/README.md

```diff
@@ -154,17 +154,19 @@ and upload these zip packages to the download section of the github
 release. For example:
 
 ```
-$ ./build-zip.sh 3.6.0
+$ ./build-zip.sh protoc 3.6.0
 ```
 
-The above command will create 5 zip files:
+The above command will create 7 zip files:
 
 ```
-dist/protoc-3.0.0-beta-4-win32.zip
-dist/protoc-3.0.0-beta-4-osx-x86_32.zip
-dist/protoc-3.0.0-beta-4-osx-x86_64.zip
-dist/protoc-3.0.0-beta-4-linux-x86_32.zip
-dist/protoc-3.0.0-beta-4-linux-x86_64.zip
+dist/protoc-3.6.0-win32.zip
+dist/protoc-3.6.0-osx-x86_32.zip
+dist/protoc-3.6.0-osx-x86_64.zip
+dist/protoc-3.6.0-linux-x86_32.zip
+dist/protoc-3.6.0-linux-x86_64.zip
+dist/protoc-3.6.0-linux-aarch_64.zip
+dist/protoc-3.6.0-linux-ppcle_64.zip
```

Comment:
Do we have kokoro job to build this binary? Should we omit it here?

Reply:
This script is kind of strange because it actually fetches the binaries from the Maven Central Repository and then repackages them in zip files for the GitHub releases page. But the binaries ultimately come from Kokoro since that is how they're built before they're uploaded to Maven. Maybe we should start building these zip files through Kokoro but for now it's still a separate step.